Dockerfile

```diff
@@ -0,0 +1,86 @@
+# syntax=docker/dockerfile:1.6
```

Comment:
I suggest keeping the dockerfile yourself and not commit to this repo.

An alternative choice is to make a new subdir, put this dockerfile in, and add some comment about its propose.

Reply:
Moving it to subdirectory will require specifying build context location potentially via scripts or configuration of command aliases (for example, in docker-compose.yml or justfile).

I'll remove it from the index.